en/docs/references/pricing.md

```diff
@@ -0,0 +1,227 @@
+# Pricing
+
+## Introduction
+
+Choreo is a Digital Platform as a Service that helps you develop integration solutions faster by reducing the coding effort with a low-code development platform. You can use Choreo by paying only for your usage: nothing more, nothing less. Choreo has payment plans for all types of users. You can enjoy the free tier with limitations and then move on to the Pay-As-You-Go model if you use more than 5,000 [steps](#what-is-a-step) per month. If you plan to use Choreo significantly, you can also opt for an Enterprise plan and enjoy special discount rates.
```

Comment:
Is "extensively" a better word to use in this context instead of "significantly"?

Reply:
This word was suggested at the review round we had with the leadership.

Reply:
Used extensively